test/integration/src/main/java/io/pravega/test/integration/demo/ControllerWrapper.java

```diff
@@ -97,11 +97,11 @@ public ControllerWrapper(final String connectionString, final boolean disableEve
                              final int controllerPort, final String serviceHost, final int servicePort,
                              final int containerCount, int restPort,
                              boolean enableAuth, String passwordAuthHandlerInputFilePath,
-                             String tokenSigningKey, int accessTokenTtlInSeconds) {
+                             String tokenSigningKey, int accessTokenTtl) {
```

Comment:
Put `seconds` back. You do not have a `Duration` that reflects time; you have an `int` where you need to tell the caller somehow how to interpret that value. Adding the `Seconds` suffix indicates that this should be time expressed in seconds.